repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Recover.scala

```diff
@@ -0,0 +1,65 @@
+package com.snowplowanalytics.snowplow.storage.bigquery.repeater
+
+import fs2.Stream
+import cats.effect.Concurrent
+import cats.effect.Timer
+import blobstore.Path
+import blobstore.implicits.GetOps
+import io.circe.JsonObject
+import java.time.Instant
+import java.util.UUID
+import io.circe.generic.auto._, io.circe.parser._
+import cats.effect.Sync
+import cats.syntax.all._
+
+object Recover {
+
+  //TODO: hardcode proper GCS path
+  val DeadQueuePath = Path("gs://sp-storage-loader-failed-inserts-dev1-com_snplow_eng_gcp/dead_queue")
+
+  def recoverFailedInserts[F[_]: Timer: Concurrent](resources: Resources[F]): Stream[F, Unit] =
+    resources
+      .store
+      .list(DeadQueuePath)
+      .evalMap(resources.store.getContents)
```

Comment:
I'm not familiar with `Store.getContents`, do all the data need to fit into memory ? If yes, are we sure that data is not too big?

Reply:
I asumed that every single file should fit in memory without problems. Is that a wrong assumption?

Reply:
Maybe you can read the files line by line to be sure?

Reply:
Yes, I can definitely. I just thought these files are pretty small. Thanks you both!